Zip Handler Web App.py

```diff
@@ -0,0 +1,33 @@
+from aiohttp import web
+import zipfile
+
+
+async def handle(request):
+    return web.Response(text='''<!DOCTYPE html>
+<html>
+<body>
+    <h1>Web app to extract files from the zip file</h1>
+    <h3>Upload a zip file</h3>
+    <form enctype="multipart/form-data" action="Filename" method="post">
+        <p><input type="file" name="filename" /></p>
+        <p><input type="submit" value="Extract"/></p>
+    </form>    
+</body>
+</html>''', content_type='text/html')
+
+
+async def handle1(request):
+    data = await request.post()
+    zip_file = data['filename'].file
+    with zipfile.ZipFile(zip_file, 'r') as z:
+        extracted_files = z.namelist()
+        html = "<ul>"
```

Comment:
Try not to have html script included in the python script, have separate html files.